platform/platform-app/src/main/resources/features.xml

```diff
@@ -34,10 +34,12 @@
 
     <repository>mvn:org.apache.karaf.decanter/apache-karaf-decanter/${decanter.version}/xml/features</repository>
 
+    <repository>mvn:org.ops4j.pax.cdi/pax-cdi-features/${pax.cdi.version}/xml/features</repository>
```

Comment:
❓ Why was this added? What features do we need from it? Can we get away with not including the entire repository? Why doe we need the `pax-cdi` feature on Line 840?